pkgs/by-name/to/tomboy-ng/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fpc,
+  lazarus,
+  autoPatchelfHook,
+
+  glib,
+  cairo,
+  pango,
+  gtk2,
+  gdk-pixbuf,
+  at-spi2-atk,
+  xorg,
+  libnotify,
+
+  nix-update-script,
+}:
+stdenv.mkDerivation (
+  finalAttrs:
+  let
+    src = fetchFromGitHub {
+      owner = "tomboy-notes";
+      repo = "tomboy-ng";
+      rev = "v${finalAttrs.version}";
+      hash = "sha256-2zU61/ulFJa1337SqbUQFi/Hv5s+wM3h3t/nIR/3XvM=";
+    };
+    kcontrols = fetchFromGitHub {
+      owner = "davidbannon";
+      repo = "KControls";
+      rev = "4b74f50599544aa05d76385c21795ca9026e9657";
+      hash = "sha256-AHpcbt5v9Y/YG9MZ/zCLLH1Pfryv0zH8UFCgY/RqrdQ=";
+      name = "kcontrols";
+    };
+  in
+  {
+    pname = "tomboy-ng";
+    version = "0.39";
+
+    srcs = [
+      src
+      kcontrols
+    ];
+
+    sourceRoot = src.name;
+
+    nativeBuildInputs = [
+      fpc
+      lazarus
+      autoPatchelfHook
+    ];
+
+    buildInputs = [
+      glib
+      cairo
+      pango
+      gtk2
+      gdk-pixbuf
+      at-spi2-atk
+      xorg.libX11
+      libnotify
+    ];
+
+    patches = [ ./simplify-build-script.patch ];
+
+    preBuild = "ln -s ../kcontrols kcontrols";
+
+    makeFlags = [ "PREFIX=${placeholder "out"}" ];
+
+    passthru.updateScript = nix-update-script { };
```

Comment:
```suggestion
    passthru = {
      updateScript = nix-update-script { };
      kcontrols = fetchFromGitHub {
        owner = "davidbannon";
        repo = "KControls";
        rev = "4b74f50599544aa05d76385c21795ca9026e9657";
        hash = "sha256-AHpcbt5v9Y/YG9MZ/zCLLH1Pfryv0zH8UFCgY/RqrdQ=";
      };
    };
```

Reply:
There is no reason to include `kcontrols` in `passthru`, as it is directly used by the derivation. I think it should just be a regular attribute of the derivation.